docs/modules/ROOT/pages/dapp.adoc

```diff
@@ -159,14 +159,17 @@ $ npm i react-app-rewired
 
 To use `react-app-rewired`, we now need to change the `package.json` scripts.  In `package.json` in the `client` directory, update the scripts to use `react-app-rewired` so that they looks as follows:
 
-[source,javascript]
+[source,diff]
```

Comment:
When using `diff`, it's better to start all lines that don't have a `+` or `-` with an extra space, so that we don't screw up the indentation.

Here for example, they end up with 3 levels of indentation instead of 2
![image](https://user-images.githubusercontent.com/2530770/72356200-9ab27580-36c7-11ea-9cea-7e2a3a828a57.png)